messages/deployCommand.json

```diff
@@ -0,0 +1,3 @@
+{
+  "resultsDirMissing": "You must supply outputdir when choosing to produce code coverage or junit reports."
```

Comment:
@jshackell-sfdc should the name `outputdir` be changed to `resultsdir` ?

Reply:
```suggestion
  "resultsDirMissing": "You must supply the --resultsdir parameter when choosing to produce code coverage or junit reports."
```

Reply:
Good catch, @maggiben !